### ISSUE TITLE
Playback notification appears on primary monitor only

### ISSUE THREAD

**Issue by Keith94** [digimezzo/dopamine-windows#317]
If Dopamine window is moved to non-primary display, the notification should appear on that same display. Currently the notification appears on the primary display only.

**Comment by Rumsey-** [comment]
I agree.  It would also be awesome if you could choose which monitor you could send the notification too.
